src/dashboard/src/media/js/archival_storage/archival_storage_search.js

```diff
@@ -49,14 +49,15 @@ $(document).ready(function() {
 
   // default field name field
   search.addSelect('field', {title: 'field name'}, {
-    ''             : 'Any',
-    'FILEUUID'     : 'File UUID',
-    'filePath'     : 'File path',
-    'fileExtension': 'File extension',
-    'AIPUUID'      : 'AIP UUID',
-    'sipName'      : 'AIP name',
-    'isPartOf'     : 'Part of AIC',
-    'AICID'        : 'AIC Identifier',
+    ''                : 'Any',
+    'FILEUUID'        : 'File UUID',
+    'filePath'        : 'File path',
+    'fileExtension'   : 'File extension',
+    'AIPUUID'         : 'AIP UUID',
+    'sipName'         : 'AIP name',
+    'isPartOf'        : 'Part of AIC',
+    'AICID'           : 'AIC Identifier',
+    'transferMetadata': 'Transfer metadata',
```

Comment:
Yeah, re-adjusting a bunch of lines because you added a long one is annoying, and having one be misaligned is also annoying. Not a strong preference.